lib/catalog/owid/catalog/meta.py

```diff
@@ -106,28 +106,31 @@ def update(self, **kwargs: Dict[str, Any]) -> None:
 @dataclass_json
 @dataclass
 class Origin:
-    # Dataset title written by OWID (without a year)
-    dataset_title_owid: str
-    # Dataset title written by producer (without a year)
-    dataset_title_producer: Optional[str] = None
-    # Our description of the dataset
-    dataset_description_owid: Optional[str] = None
-    # The description for this dataset used by the producer
-    dataset_description_producer: Optional[str] = None
-    # The name of the institution (without a year) or the main authors of the paper
-    producer: Optional[str] = None
-    # The full citation that the producer asks for
-    citation_producer: Optional[str] = None
+    # Producer name
+    # Name of the institution or the author(s) that produced the data product.
+    producer: str
+    # Title of the original data product
+    title: str
+    # Description of the data product
+    description: Optional[str] = None
+    # Title of the snapshot
+    # Subset of data that we extract from the data product which in most cases will coincide with the title of the data product.
+    title_snapshot: Optional[str] = None
+    # Description of the snapshot
+    # Subset of data that we extract from the data product) which in most cases will coincide with the description of the data product.
```

Comment:
There's a spurious ")". Also, this explanation is not true: The `description_snapshot` will be empty in most cases. Only when the data product and the snapshot do not coincide, the `description_snapshot` will contain **additional** information to the description of the data product. We decided that, to avoid redundance, `description_snapshot` should not contain `description`.

Reply:
If you prefer I can handle these things. I'm just making sure we agree on the underlying logic.